pyfr/integrators/base.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, backend, rallocs, mesh, initsoln, cfg):
         self.rallocs = rallocs
         self.isrestart = initsoln is not None
         self.cfg = cfg
+        cfgsect = 'solver-time-integrator'
```

Comment:
Do we need to declare this?

Reply:
Line 64 otherwise exceeds the line limit.

Reply:
You can just line break it, if it is only one line causing an issue.